meta-facebook/mfz-wc/src/platform/plat_sensor_table.c

```diff
@@ -255,6 +255,26 @@ void check_vr_type(uint8_t index)
 
 void pal_fix_sensor_config()
 {
+	uint8_t sensor_count = ARRAY_SIZE(plat_sensor_config);
```

Comment:
why?

Reply:
After discussion with the reviewer, this code is not proper for now to MFZ code. So I removed it.